src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div flat id="list-people-roles" class="mt-0">
+  <div id="list-people-roles" class="mt-0">
```

Comment:
I'd remove the mt-0 as it's probably needed and may conflict with classes for this component set in the parent view.

Reply:
Removed 👍